cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/realtime/sinks/RealtimeCubeSink.java

```diff
@@ -47,29 +52,49 @@
  * {@link co.cask.cdap.api.dataset.lib.cube.CubeFact} the
  * mapping configuration is required, as per {@link StructuredRecordToCubeFact} documentation.
  */
-// todo: add unit-test once CDAP-2156 is resolved
+@Plugin(type = "sink")
+@Name("RealtimeCube")
```

Comment:
This should be RealtimeCube and the batch one Cube right ? 
Thats what we have for others.